pyproject.toml

```diff
@@ -96,7 +96,7 @@ trino = { version = ">=0.319,<1", optional = true, extras = ["sqlalchemy"] }
 [tool.poetry.group.dev.dependencies]
 black = ">=22.1.0,<24"
 google-cloud-storage = ">=2.7.0,<3"
-ipython = ">=7.27.0,<9"
+ipython = ">8.6.0,<9"
```

Comment:
I guess 8.6.0 is not okay? Maybe we should have the `>=` version for consistency with the rest of the specs and for clarity on which specific lower bound works?